public/static/locales/en/projectDetails.json

```diff
@@ -1,5 +1,10 @@
 {
   "ProjectDetails": {
+    "allProjects": "All (<projectCountContainer>{noOfProjects}</projectCountContainer>)",
+    "searchProject": "Search Project",
+    "topProjects": "Top Projects (<projectCountContainer>{noOfProjects}</projectCountContainer>)",
+    "list": "List",
+    "map": "Map",
```

Comment:
Perhaps these translations should be in `allProjects.json` instead of `projectDetails.json`.